### PR TITLE
Add missing required/optional attributes

### PR DESCRIPTION
**Pull request by jenssegers**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -611,10 +610,10 @@ Update a contact.
         + birthdate: `1989-08-19` (string, optional)
         + remarks: `Met at expo` (string, optional)
         + tags: expo, upsell (array[string], optional)
-        + custom_fields (array)
+        + custom_fields (array, optional)
             + (object)
-                + field_id: `9b48244c-3be9-4f8e-b4d4-da18ffedfb6b` (string)
-                + value: `092980616`
+                + field_id: `9b48244c-3be9-4f8e-b4d4-da18ffedfb6b` (string, required)
+                + value (required)
```
Comment (mrnnn) [inline apiary.apib]: The actual value is gone, is that on purpose?

File: apiary.apib [hunk 2]
```diff
@@ -662,8 +661,8 @@ Link a contact to a company.
     + Attributes (object)
         + id: `75596038-b9c8-4a37-969d-61059e300a28` (string, required)
         + company_id: `4da20f00-0628-4336-b885-aa90e580dd85` (string, required)
-        + position: `CEO` (string)
-        + decision_maker: true (boolean)
+        + position: `CEO` (string, optional)
+        + decision_maker: true (boolean, optional)
```
Comment (mrnnn) [inline apiary.apib]: NTH: document default value in this case?

File: src/02-crm/companies.apib [hunk 3]
```diff
@@ -38,7 +38,7 @@ Get a list of companies.
                 + vat_number: `BE0899623035` (string)
                 + local_business_number: `63326426` (string)
                 + email: `info@piedpiper.eu` (string)
-                + invoice_email: `invoicing@example.com` (string, optional)
+                + invoice_email: `invoicing@example.com` (string)
```
Comment (mrnnn) [inline src/02-crm/companies.apib]: haha, indeed

 :ok: by @mrnnn on 16/11/2017 14:56:47

File: apiary.apib (hunk 2 above)
Comment (stivni) [reply apiary.apib]: I wrote down to check this in a follow up PR

File: src/04-invoicing/invoices.apib [hunk 4]
```diff
@@ -14,7 +14,7 @@ List invoices
                     + Members
                         + contact
                         + company
-                + id: `e592218d-c0de-4153-a39a-eb93ff5f1ccc` (string)
+                + id: `e592218d-c0de-4153-a39a-eb93ff5f1ccc` (string, required)
```
Comment (stivni) [inline src/04-invoicing/invoices.apib]: NTH : leave it optional, then you can filter on _all contacts_ eg.

File: src/99-other/attachments.apib [hunk 5]
```diff
@@ -113,17 +113,17 @@ Add a file, by first posting its meta-data and afterwards PUTting the contents t
 
     + Attributes (object)
         + path: `design/v1.0/logo.jpg` (string, optional)
-        + size: `3355` (number) - In bytes
-        + mime_type (string)
+        + size: `3355` (number, optional) - In bytes
```
Comment (stivni) [inline src/99-other/attachments.apib]: Q : why optional?

File: src/99-other/attachments.apib [hunk 6]
```diff
@@ -113,17 +113,17 @@ Add a file, by first posting its meta-data and afterwards PUTting the contents t
...
+        + mime_type (string, optional)
```
Comment (stivni) [inline src/99-other/attachments.apib]: Q : why optional?

**Pull request merged**